addons/survey/static/src/js/survey_form.js

```diff
@@ -297,6 +328,26 @@ publicWidget.registry.SurveyFormWidget = publicWidget.Widget.extend({
     // INIT FIELDS TOOLS
     // -------------------------------------------------------------------------
 
+    _updateBreadcrumb: function ($target) {
+        var $breadcrumb = this.$('.breadcrumb');
+        var activeFound = false;
+        $breadcrumb.find('.breadcrumb-item').each(function () {
+            var span = "<span>"+ $(this).data('pageTitle') +"</span>";
+            if ($target && $(this).data('pageId') === $target.data('pageId')) {
+                $(this).addClass("active");
+                activeFound = true;
+            } else {
+                $(this).removeClass("active");
+            }
+            if (activeFound) {
+                $(this).html(span);
+            } else {
+                $(this).html("<a href='#'>" + span + "</a>");
+            }
+        });
+        $breadcrumb.toggleClass('d-none', $target ? false : true);
+    },
+    
```

Comment:
Please use a QWeb template with parameters to build html like this.

Also, be careful with "html" method call, this introduces a XSS attack vector by allowing to put JS script in your page titles.